src/ssa_ir/converter/expr_converter.rs

```diff
@@ -0,0 +1,1014 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use core::panic;
+
+use solang_parser::pt::Loc;
+
+use crate::codegen::Expression;
+use crate::sema::ast;
+use crate::ssa_ir::converter::Converter;
+use crate::ssa_ir::expr::{BinaryOperator, Expr, Operand, UnaryOperator};
+use crate::ssa_ir::insn::Insn;
+use crate::ssa_ir::vartable::Vartable;
+
+impl Converter<'_> {
+    pub(crate) fn convert_expression(
```

Comment:
How about calling is Lowering -- it is more specific than converting which could mean anything